lib/engine/game/base.rb

```diff
@@ -704,9 +734,7 @@ def next_turn!
       end
 
       def action_from_h(h)
-        Object
-          .const_get("Engine::Action::#{Action::Base.type(h['type'])}")
-          .from_h(h, self)
+        Engine::Action::Base.action_from_h(h, self)
```

Comment:
Should we remove this method and replace its calls with just creating the action where it's used?

Reply:
good call